### DISCUSSION TITLE
Remove GitHub's changelog from my homepage

### DISCUSSION THREAD

**Discussion by mrjones2014** [community/community#16446] [Other Feature Feedback, Questions, & Ideas]
It's taking up half my vertical screen space with info that, frankly, I just don't care about (sorry) and pushing down the "Explore Repositories" which I _do_ care about. 

Either remove it or put it below "Explore Repositories" 

![image](https://user-images.githubusercontent.com/8648891/168095059-325ccfcb-5e41-4e2f-94b6-5ac013ffa90e.png)

**Comment by mrjones2014** [comment]
I've just blocked it with uBlock Origin for now but please please add an option to hide this so I don't have to add a uBlock Origin rule every time I use a device, I also can't block it on, for example, iPad.

**Comment by sunxfancy** [comment]
I feel no one care about the changelog of Github. It's better to remove it and show more about explore repositories.

**Reply by mrjones2014** [reply]
Yeah I’m really not sure who asked for or wanted this…

**Comment by tausifcreates** [comment]
Stupidity unbound
